src/pages/VenuePage/VenuePage.tsx

```diff
@@ -66,12 +150,22 @@ export const VenuePage: React.FC = () => {
 
   // @debt Remove this once we replace currentVenue with currentVenueNG or similar across all descendant components
   useConnectCurrentVenue();
-  const venue = useSelector(currentVenueSelector);
-  const venueRequestStatus = useSelector(isCurrentVenueRequestedSelector);
+  const postVenue = useSelector(currentVenueSelector);
+  const postVenueLoaded = useSelector(isCurrentVenueRequestedSelector);
 
   useConnectCurrentEvent();
-  const currentEvent = useSelector(currentEventSelector);
-  const eventRequestStatus = useSelector(isCurrentEventRequestedSelector);
+  const postEvents = useSelector(currentEventSelector);
+  const postEventsLoaded = useSelector(isCurrentEventRequestedSelector);
+
+  if (preError) {
+    console.warn("VenuePage()", "preload failed:", preError.message);
+  }
+
+  const venue = (preError ? undefined : preVenue) ?? postVenue;
+  const isVenueLoaded = (preError ? undefined : preLoaded) ?? postVenueLoaded;
+
+  const events = (preError ? undefined : preEvents) ?? postEvents;
+  const isEventLoaded = (preError ? undefined : preLoaded) ?? postEventsLoaded;
```

Comment:
Why are these renamed? There's no point having both of this and the 'fetch via backend function' code. I suggest we just stick to this code as we already had it if it's fast enough now that the excess data fetching has been slimmed down.

Reply:
Same as https://github.com/sparkletown/sparkle/pull/1713#discussion_r666013755